### ISSUE TITLE
🐛 Bug: Fetching model eos54ij got "Not a directory" error

### ISSUE THREAD

**Issue by RoseZhen2022** [ersilia-os/ersilia#345]
### Describe the bug.

I have fetch model eos54ij multiple times and tried to fetch it with or without export log, both got same error:
NotADirectoryError: [Errno 20] Not a directory

### Describe the steps to reproduce the behavior

1. ersilia -v fetch eos54ij | tee -a eos54ij.log 2>&1
2. ersilia -v fetch eos54ij

### Expected behavior.

Hope to fetch the model and do next step

### Screenshots.

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/115380880/195975314-6ad90614-4503-477c-8958-ba327575fda8.png"><img width="1243" alt="image" src="https://user-images.githubusercontent.com/115380880/195975326-22e12dd0-7c4f-4c29-ab33-aafc515ec9c6.png">

### Operating environment

macOS Monterey

### Additional context

The log file created is nothing inside, so I didn't put the log file.

**Comment by GemmaTuron** [comment]
Hello @RoseZhen2022 

I just run it on my linux machine without problems:

17:30:30 | DEBUG    | [{'input': {'key': 'LUHMMHZLDLBAKX-UHFFFAOYSA-N', 'input': 'CC1C2C(CC3(C=CC(=O)C(=C3C2OC1=O)C)C)O', 'text': 'CC1C2C(CC3(C=CC(=O)C(=C3C2OC1=O)C)C)O'}, 'output': {'outcomes': [None, None, 0.8, 0.0, 0.2, 0.26]}}, {'input': {'key': 'QRXWMOHMRWLFEY-UHFFFAOYSA-N', 'input': 'C1=CN=CC=C1C(=O)NN', 'text': 'C1=CN=CC=C1C(=O)NN'}, 'output': {'outcomes': [None, None, 0.9, 0.0, 0.25, 0.26]}}]
17:30:30 | DEBUG    | Latest meta: {'outcomes': None}
17:30:30 | DEBUG    | outcomes : {'type': 'mixed_array', 'shape': (6,)}
17:30:30 | DEBUG    | Meta k: None
17:30:30 | DEBUG    | Schema: {'input': {'key': {'type': 'string'}, 'input': {'type': 'string'}, 'text': {'type': 'string'}}, 'output': {'outcomes': {'type': 'mixed_array', 'shape': (6,), 'meta': None}}}
17:30:30 | DEBUG    | {'input': {'key': {'type': 'string'}, 'input': {'type': 'string'}, 'text': {'type': 'string'}}, 'output': {'outcomes': {'type': 'mixed_array', 'shape': (6,), 'meta': None}}}
17:30:30 | DEBUG    | API schema saved at /home/gturon/eos/dest/eos54ij/api_schema.json
17:30:30 | INFO     | Fetching eos54ij done successfully: 0:02:53.585336
👍 Model eos54ij fetched successfully!

Can you let me know which:
- Conda version
- Python version
you are using?

**Comment by GemmaTuron** [comment]
Also, the "Not a Directory" error seems like you are running things from a location that ersilia is unable to find. Can you try to run the command from your desktop?

**Comment by GemmaTuron** [comment]
Hi @RoseZhen2022 
Have you been able to check my suggestions?

**Comment by RoseZhen2022** [comment]
Hi @GemmaTuron  Sorry for the late reply. As the operating system I used before has many problems. So recently I changed to another system Ubuntu 22.04.1 LTS and set the new Erisilia working environment. Using this new system, I have successfully predicted 3 of 4 models you assigned me before and I have already added the output csv files to drive. However, only this model eos54ij was fetched failed with "EmptyOutputError". And here is the log of this model.
[eos54ij.log](https://github.com/ersilia-os/ersilia/files/9814659/eos54ij.log)
